templates/admin/view_rta_regenerate.php

```diff
@@ -70,6 +70,9 @@
         </div>
       </div>
 
+        <a href="https://help.shortpixel.com/article/233-quick-guide-to-using-regenerate-thumbnails-advanced-settings"
+           class="shortpixel-help-link" target="_blank">
+            <span class="dashicons dashicons-editor-help"></span>Read more</a>
```

Comment:
In this implementation the question mark gets underlined as well and it needs a `text-decoration: none;` and `margin-top: -1px;`, just like we have in SPIO:

https://github.com/short-pixel-optimizer/shortpixel-image-optimiser/blob/master/res/css/short-pixel.css#L656

I advise you to add similar rules directly in https://github.com/short-pixel-optimizer/regenerate-thumbnails-advanced-rewrite/blob/master/css/rta.css if you want to go without altering the scss files.

Reply:
Thanks for the notice.  I have made the change and tested. The underline is not available on :active, nor on :hover